docs/organizations/using-a-coding-standard.md

```diff
@@ -52,7 +52,7 @@ To create a coding standard for your organization:
 
 ## Setting a coding standard as default {: id="set-default"}
 
-New repositories in your organization will automatically follow the default coding standard.
+New repositories in your organization automatically follow the default coding standard.
```

Comment:
There's an opportunity here to clarify what currently happens when the single coding standard isn't defined as "default" - which is that new repositories will use the "Codacy defaults".

This could then change depending on future product changes.

Reply:
```suggestion
New repositories in your organization automatically follow the default coding standard or Codacy defaults if no default coding standard is set.
```